test/setup_bundler_test.rb

```diff
@@ -186,6 +215,7 @@ def test_creates_custom_bundle_if_no_gemfile
         assert_path_exists(".ruby-lsp")
         assert_path_exists(".ruby-lsp/Gemfile")
         assert_match("ruby-lsp", File.read(".ruby-lsp/Gemfile"))
+        # assert_match("ruby-lsp-rails", File.read(".ruby-lsp/Gemfile"))
```

Comment:
This shouldn't be in the custom bundle if there's no top level Gemfile (because Rails has no way to be there).
✂️